e2e/process-services-cloud/start-task/start-task-group-cloud.e2e.ts

```diff
@@ -0,0 +1,194 @@
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { browser } from 'protractor';
+import { NavigationBarPage } from '../../pages/adf/navigationBarPage';
+import { TasksCloudDemoPage } from '../../pages/adf/demo-shell/process-services/tasksCloudDemoPage';
+import {
+    LoginSSOPage, AppListCloudPage, StringUtil, GroupCloudComponentPage,
+    StartTasksCloudPage, PeopleCloudComponentPage, TasksService, ApiService, IdentityService, SettingsPage, GroupIdentityService
+} from '@alfresco/adf-testing';
+import resources = require('../../util/resources');
+
+describe('Start Task - Group Cloud Component', () => {
+
+    const loginSSOPage = new LoginSSOPage();
+    const navigationBarPage = new NavigationBarPage();
+    const appListCloudComponent = new AppListCloudPage();
+    const tasksCloudDemoPage = new TasksCloudDemoPage();
+    const startTask = new StartTasksCloudPage();
+    const peopleCloudComponent = new PeopleCloudComponentPage();
+    const settingsPage = new SettingsPage();
+    const apiService = new ApiService(
+        browser.params.config.oauth2.clientId,
+        browser.params.config.bpmHost, browser.params.config.oauth2.host, browser.params.config.providers
+    );
+    const groupCloud = new GroupCloudComponentPage();
+
+    const unassignedTaskName = StringUtil.generateRandomString(5);
+    let apsUser, testUser, hrGroup, testGroup;
+    const simpleApp = resources.ACTIVITI7_APPS.SIMPLE_APP.name;
+
+    let identityService: IdentityService;
+    let groupIdentityService: GroupIdentityService;
+
+    beforeAll(async (done) => {
+        await apiService.login(browser.params.identityAdmin.email, browser.params.identityAdmin.password);
+
+        identityService = new IdentityService(apiService);
+        groupIdentityService = new GroupIdentityService(apiService);
+        testUser = await identityService.createIdentityUser();
+        apsUser = await identityService.createIdentityUser();
+
+        hrGroup = await groupIdentityService.getGroupInfoByGroupName('hr');
+        testGroup = await groupIdentityService.getGroupInfoByGroupName('testgroup');
+        await identityService.addUserToGroup(testUser.idIdentityService, testGroup.id);
+        await identityService.addUserToGroup(apsUser.idIdentityService, hrGroup.id);
+
+        await settingsPage.setProviderBpmSso(
+            browser.params.config.bpmHost,
+            browser.params.config.oauth2.host,
+            browser.params.config.identityHost);
+        done();
+    });
+
+    afterAll(async (done) => {
+        try {
+            await apiService.login(testUser.email, testUser.password);
+            const tasksService = new TasksService(apiService);
+
+            const taskId = await tasksService.getTaskId(unassignedTaskName, simpleApp);
+            if (taskId) {
```

Comment:
I don't see a need for this If logic here. If we are expecting a task to be created before or during the tests, then it can be deleted.